### ISSUE TITLE
aws_route53_zone : panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by meriouma** [terraform-providers/terraform-provider-aws#8370]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.11.13

### Affected Resource(s)

* aws_route53_zone

### Terraform Configuration Files

```hcl
variable "region" {
    default = "us-east-1"
}

variable "hostname" {
    default = "localhost"
}

provider "aws" {
    region = "${var.region}"
    access_key = "unused"
    secret_key = "unused"

    endpoints {
        apigateway = "http://${var.hostname}:4567"
        cloudformation = "http://${var.hostname}:4581"
        cloudwatch = "http://${var.hostname}:4582"
        dynamodb = "http://${var.hostname}:4569"
        es = "http://${var.hostname}:4578"
        firehose = "http://${var.hostname}:4573"
        iam = "http://${var.hostname}:4593"
        kinesis = "http://${var.hostname}:4568"
        lambda = "http://${var.hostname}:4574"
        route53 = "http://${var.hostname}:4580"
        redshift = "http://${var.hostname}:4577"
        s3 = "http://${var.hostname}:4572"
        secretsmanager = "http://${var.hostname}:4584"
        ses = "http://${var.hostname}:4579"
        sns = "http://${var.hostname}:4575"
        sqs = "http://${var.hostname}:4576"
        ssm = "http://${var.hostname}:4583"
        stepfunctions = "http://${var.hostname}:4585"
        sts = "http://${var.hostname}:4592"
    }

    s3_force_path_style = true
    skip_credentials_validation = true
    skip_metadata_api_check = true
    skip_requesting_account_id = true
}

resource "aws_route53_zone" "domain" {
    name = "somedomain.ca"
}
```

### Panic Output

https://gist.github.com/meriouma/21604782c76d1dfcf824a0e5270c8802

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

Using localstack

**Comment by nywilken** [comment]
@meriouma thanks for reporting this issue. I'm pulling out the log line that highlights where the resource is throwing the panic error. 

```
2019-04-18T05:32:53.228Z [DEBUG] plugin.terraform-provider-aws_v2.6.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_route53_zone.go:144
```

**Comment by linux2647** [comment]
This may be caused by the underlying localstack service not returning a ChangeInfo object as part of the CreateHostedZoneRespsonse object.  moto's route53 module [appears to be missing said object](https://github.com/spulec/moto/blob/75d9707e8ebad300b1a79ac6fc23ed1ccc65d9b3/moto/route53/responses.py#L294-L311) from its response.

**Comment by bflad** [comment]
Fix submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/12634

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.60.0 of the Terraform AWS Provider, later this week. 👍
